prompttools/testing/test_response_quality.py

```diff
@@ -0,0 +1,51 @@
+from prompttools.structures.prompt import PromptTest
```

Comment:
The 'testing' directory is being used for prompttest logic. For this, which is an example of a test, it would be best in the `examples/` directory.